registration/tests.py

```diff
@@ -0,0 +1,165 @@
+from django.contrib.auth import get_user_model
+from django.core.management import call_command
+from django.core.management.base import CommandError
+from django.test import TestCase
+from io import StringIO
+
+import os
+import sys
+
+from participant.models import Troop
+from registration.management.commands import create_users
+
+
+class ImportUsersTestCase(TestCase):
+    def tearDown(self):
+        try:
+            os.remove('test.csv')
+        except OSError:
+            pass
+
+    def test_something(self):
+        sys.stdout = open(os.devnull, 'w')
+        sys.stderr = open(os.devnull, 'w')
```

Comment:
instead of setting `stdout` and `stderr` on `sys`, you could probably set it directly on `cmd`, no? (I am not sure something like that works for python)

```python
        cmd = create_users.Command()
        cmd.stdout = StringIO()
        cmd.stderr = StringIO()
        cmd.import_lines(lines)
```

Reply:
Indeed!